src/scripts/eos-analysis

```diff
@@ -62,6 +62,7 @@ Lists the named prior PDFs defined within the scope of this analysis file.
     )
     parser_list_priors.set_defaults(cmd = cmd_list_priors)
 
+
```

Comment:
I added a couple of new lines for readability.

Reply:
Thanks!